kernel/src/debug.rs

```diff
@@ -222,7 +222,7 @@ pub struct DebugWriter {
 /// needed so the debug!() macros have a reference to the object to use.
 static mut DEBUG_WRITER: Option<&'static mut DebugWriterWrapper> = None;
 
-pub static mut OUTPUT_BUF: [u8; 64] = [0; 64];
+pub static mut OUTPUT_BUF: [u8; 1024] = [0; 1024];
```

Comment:
This should be reverted before merging.

Reply:
Hey brad, this is still incomplete, I was hoping people would use this to help me debug the multi-send issue. There are still tons of debug!() statements that need to be removed, and documentation to be added.